src/components/Data/Documents/Page.vue

```diff
@@ -26,16 +28,49 @@
             }"
             >Create New Document</b-button
           >
+          <b-dropdown
+            class="ml-1"
+            data-cy="Refresh-dropdown"
+            split
+            variant="outline-primary"
+            :title="
+              autoSync
+                ? 'Documents are updated in real-time'
+                : 'Refresh the list to apply pending notifications'
```

Comment:
`notifications` could be confusing for users (they are used to Instagram notifications)
```suggestion
                : 'Refresh the list to apply pending changes'
```